src/patternfly/components/Progress/progress.scss

```diff
@@ -1,40 +1,39 @@
+  --#{$progress}__status--Gap: var(--pf-t--global--spacer--sm);
+  --#{$progress}__status-icon--Color: var(--pf-t--global--icon--color--regular);
   --#{$progress}__indicator--Height: var(--#{$progress}__bar--Height);
-  --#{$progress}__indicator--BackgroundColor: var(--#{$progress}__bar--before--BackgroundColor);
-  --#{$progress}__helper-text--MarginTop: calc(var(--#{$pf-global}--spacer--xs) - var(--#{$progress}--GridGap));
-
+  --#{$progress}__indicator--BackgroundColor: var(--pf-t--global--color--brand--default);
+  --#{$progress}__helper-text--MarginTop: calc(var(--pf-t--global--spacer--sm) - var(--#{$progress}--GridGap));
+  
   // Modifier variables
-  --#{$progress}--m-success__bar--BackgroundColor: var(--#{$pf-global}--success-color--100);
-  --#{$progress}--m-warning__bar--BackgroundColor: var(--#{$pf-global}--warning-color--100);
-  --#{$progress}--m-danger__bar--BackgroundColor: var(--#{$pf-global}--danger-color--100);
-  --#{$progress}--m-success__status-icon--Color: var(--#{$pf-global}--success-color--100);
-  --#{$progress}--m-warning__status-icon--Color: var(--#{$pf-global}--warning-color--100);
-  --#{$progress}--m-danger__status-icon--Color: var(--#{$pf-global}--danger-color--100);
-  --#{$progress}--m-inside__indicator--MinWidth: var(--#{$pf-global}--spacer--xl);
-  --#{$progress}--m-inside__measure--Color: var(--#{$pf-global}--Color--light-100); // the measure needs to be white when moved inside the progress bar indicator
-  --#{$progress}--m-success--m-inside__measure--Color: var(--#{$pf-global}--Color--light-100); // remove at breaking change
-  --#{$progress}--m-warning--m-inside__measure--Color: var(--#{$pf-global}--Color--dark-100); // the measure still needs to be dark when in warning state
-  --#{$progress}--m-inside__measure--FontSize: var(--#{$pf-global}--FontSize--sm);
-  --#{$progress}--m-outside__measure--FontSize: var(--#{$pf-global}--FontSize--sm);
-  --#{$progress}--m-sm__bar--Height: var(--#{$pf-global}--spacer--sm);
-  --#{$progress}--m-sm__description--FontSize: var(--#{$pf-global}--FontSize--sm);
-  --#{$progress}--m-sm__measure--FontSize: var(--#{$pf-global}--FontSize--sm);
-  --#{$progress}--m-lg__bar--Height: var(--#{$pf-global}--spacer--lg);
-
-  // Component styles
+  --#{$progress}--m-success__bar--BackgroundColor: var(--pf-t--global--color--status--success--default);
+  --#{$progress}--m-warning__bar--BackgroundColor: var(--pf-t--global--color--status--warning--default);
+  --#{$progress}--m-danger__bar--BackgroundColor: var(--pf-t--global--color--status--danger--default);
+  --#{$progress}--m-success__status-icon--Color: var(--pf-t--global--color--status--success--default);
+  --#{$progress}--m-warning__status-icon--Color: var(--pf-t--global--color--status--warning--default);
+  --#{$progress}--m-danger__status-icon--Color: var(--pf-t--global--color--status--danger--default);
```

Comment:
I think these should use the icon color tokens. In the designs, looks like the success icon just uses the global success color as you have here, but the warning/danger icons use the warning/danger icon color tokens - I'm assuming the success color in figma is a bug and should use the icon success color, too.